### PR TITLE
Updated LNC and Umbrel LND connector's content

### LINKED ISSUE #2258: Update LNC and Umbrel LND connrectors content
Umbrel connector content is out of date. Both copy and video there relates to previous Umbrel design:
Screenshot 2023-03-23 at 11 30 45

Let's update the copy with:

1. In your Lightning Node dashboard go to **Connect wallet**
2. Select **REST (Tor)** or **REST (Local Network)** mode
3. Copy the **lndconnect URL** and paste it below

And change the video to this screenshot:

image

---

We can update LNC connector as well.

Current version:
Screenshot 2023-03-23 at 11 40 51

Updated copy:
1. On Lightning Terminal sidebar choose **Lightning Node Connect**
2. Click **Create a new session**
3. Label the session, choose desired permissions and submit
4. **Copy pairing phrase** and paste it below

Added screenshot (insead of Alby logo):
image

### PR DESCRIPTION
**Pull request by AdarshRawat1**
### Describe the changes you have made in this PR
- Updated Umbrel connector Instructions
- Changed the video to screenshot on Umbrel connector
- Updated instructions on LNC connector
- Added screenshot in LNC connector

### Link this PR to an issue [optional]

Fixes #2258

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]
- Current Umbrel connector instructions 
image
- Updated Umbrel connector Instructions
- Changed the video to screenshot on Umbrel connector
image

- Current LNC connector -
image
- Updated instructions on LNC connector
- Added screenshot in LNC connector
image

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/components/ConnectorForm/index.tsx [hunk 1]
```diff
@@ -42,18 +44,37 @@ function ConnectorForm({
...
+          {image ? (
+            <>
+              <div className="w-16 md:w-32 mb-8 lg:w-[100%] lg:mb-0">
+                <img
+                  src={`${image}`}
+                  alt="Image"
+                  className="block dark:hidden w-full"
+                />
+                <img
+                  src={`${image}`}
```
Comment (reneaaron) [inline src/app/components/ConnectorForm/index.tsx:56]: You should be able to use `` here.

File: src/app/components/ConnectorForm/index.tsx [hunk 2]
```diff
@@ -42,18 +44,37 @@ function ConnectorForm({
...
-            />
-            <img
-              src="assets/icons/alby_logo_dark.svg"
-              alt="Alby"
-              className="hidden dark:block w-full"
-            />
-          </div>
+          {image ? (
+            <>
+              <div className="w-16 md:w-32 mb-8 lg:w-[100%] lg:mb-0">
```
Comment (reneaaron) [inline src/app/components/ConnectorForm/index.tsx:49]: You should be able to use `lg:w-full` here.

Comment (reneaaron) [reply src/app/components/ConnectorForm/index.tsx:49]: Remove the width constraints from here to improve responsive display.

**Review by reneaaron — CHANGES_REQUESTED** [review]
Thanks for your PR @AdarshRawat1! 💪

I noticed some layout issues on smaller screen sizes: 

image

The screenshot here seems to be displayed very small. I'd suggest removing the width restrictions for those kinds of images and displaying them full-width:

image


Thanks for your contribution! 🙌

File: src/app/components/ConnectorForm/index.tsx (hunk 2 above)
Comment (AdarshRawat1) [reply src/app/components/ConnectorForm/index.tsx:49]: Noted! Will work on it

**Comment by AdarshRawat1** [comment]
@reneaaron, I have made the suggested changes. Could you please review them?

**Review by reneaaron — APPROVED** [review]
Added 2 more commits where I moved the images to the CDN instead of shipping it with the extension. 

Also added some more improvements for formatting & some bugfixes, please have a look. 

tACK 

Thanks for your contribution @AdarshRawat1 :raised_hands:

**Pull request merged**